css/index.css

```diff
@@ -1 +1,1153 @@
+	/* border: 1px solid red; */ 
+  /* background: lightgray;
+  /* position: relative; */
+  height: 100px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+}
+
+.main1{
+	/* border: 1px solid yellow; */
+	/* display: inline-block; */
+	width: 47%;
+	height: 100px;
+    margin: 0 0.75%;
+
+}
+
+.main1 h5{
+	/* order: 1px solid green; */
+	
+	padding-top: 10px;
+
+}
+
+.main1 p{
+	/* order: 1px solid green; */
+	
+	font-size: 1.2rem;
+}
+.main2{
+	/* border: 1px solid purple; */
+	/* display: inline-block; */
+	width: 47%;
+	/* margin-right: 0; */
+	/* position: absolute; */
+    height: 100px;
+    margin: 0 0.75%;
+	
+}
+
+.main2 h5{
+	/* order: 1px solid purple; */
+	padding-top: 10px;
+}
+
+.main2 p{
+	/* order: 1px solid orange; */
+	font-size: 1.2rem;
+	
+}
+
+.section3{
+	/* border: 1px solid purple; */
+	/* display: block; */
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	margin-top: 10px;
+}
+
+.main3{
+	display: flex;
+	/* border: 1px solid purple; */
+	width:100%;
+	/* display: block; */
+}
+
+.middle-img{
+	/* border: 1px solid purple; */
+	/* display: block; */
+	width: 100%
+}
+
+.Section4{
+	/* border: 1px solid green; */
+	/* border: lightblue; */
+	/* display: inline-block; */
+	width: 100%;
+	height: 150px;
+	/* position: relative; */
+	display: flex;
+	flex-direction: row;
+	
+	justify-content: space-between;
+	align-items: center;
+	margin-top: 15px;
+}
+
+.Section4 .Services{
+	/* border: 1px solid purple; */
+	/* width: 150px; */
+	margin: 0 0.5%;
+	/* flex-direction: row;
+	justify-content: flex-start;
+	flex-wrap: wrap; */
+	/* right: 50px; */
+	/* display: inline-block; */
+	/* top: 0;
+	bottom: 0;
+	left:0;
+	right:150px;  */
+	/* position: absolute; */
+}
+
+.Services p{
+	/* border: 1px solid green; */
+	display: flex;
+
+}
+
+.Section4 .Products{
+	/* border: 1px solid purple; */
+	/* width: 250px;
+	height: 100px;
+	display: flex; */
+	/* flex-direction: row;
+	justify-content: flex-start;
+	flex-wrap: wrap; */
+	/* display: inline-block; */
+	/* top: 0;
+	bottom: 0;
+	left: 270px;
+	right: 160px;  */
+	/* position: absolute; */
+}
+
+
+.Products p{
+	/* border: 1px solid green; */
+	display: flex;
+}
+
+.Section4 .Vision{
+	/* border: 1px solid purple; */
+	/* width: 250px;
+	height: 100px;
+	display: flex; */
+	/* flex-direction: row;
+	justify-content: flex-start;
+	flex-wrap: wrap; */
+	/* display: inline-block;
+	top: 0;
+	bottom: 0;
+	left: 550px;
+	right: 0px;  */
+	/* position: absolute; */
+}
+
+.Vision p{
+	/* border: 1px solid green; */
+	display: flex;
+}
+
+.bottomMain{
+	/* border: 1px solid purple; */
+	border-top: 2px solid black;
+	padding-top: 15px;
+	margin-bottom: 50px;
+}
+
+.indexFooter{
+	/* border: 1px solid purple; */
+	margin-bottom: 75px;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+}
+
+.copyright{
+	/* border: 1px solid purple; */
+	display: flex;
+}
+
+.SVcontainer{
+	/* border: 1px solid red; */
+	width: 800px;
```

Comment:
Try to use percentages for `width` properties. However, you can use px for `max-width`.